src/fidesops/migrations/versions/55d61eb8ed12_add_default_policies.py

```diff
@@ -0,0 +1,215 @@
+        logger.info(
+            f"Deleting autogenerated '{AUTOGENERATED_ERASURE_KEY}' erasure policy"
+        )
+        erasure_policy.delete(db)
+
+    try:
+        logger.info(
+            f"Deleting autogenerated client: '{FIDESOPS_AUTOGENERATED_CLIENT_KEY}'"
+        )
+        client.delete(db)
+        storage = StorageConfig.filter(
+            db=db, conditions=(StorageConfig.key == FIDESOPS_AUTOGENERATED_STORAGE_KEY)
+        ).first()
+        if storage:
+            logger.info(
+                f"Deleting autogenerated local storage: '{FIDESOPS_AUTOGENERATED_STORAGE_KEY}'"
+            )
+            storage.delete(db)
+    except Exception:
+        # It's possible the client or storage config have been attached to other things
+        pass
+
+    db.close()
+
+
+def autogenerate_access_policy(
+    db: Session, client: ClientDetail, storage: StorageConfig
+) -> None:
+    """Create an autogenerated 'download' access policy, with an access rule attached,
+    targeting user.provided.identifiable data"""
+    logger.info(f"Creating autogenerated '{AUTOGENERATED_ACCCES_KEY}' policy")
+    access_policy = Policy.create_or_update(
+        db,
+        data={
+            "name": "Fidesops Autogenerated Access Policy",
+            "key": AUTOGENERATED_ACCCES_KEY,
+            "drp_action": DrpAction.access,
+            "client_id": client.id,
+        },
+    )
+
+    access_rule = Rule.create(
+        db,
+        data={
+            "name": "Fidesops Autogenerated Access Rule",
+            "action_type": ActionType.access.value,
+            "policy_id": access_policy.id,
+            "client_id": client.id,
+            "storage_destination_id": storage.id,
+        },
+    )
+
+    RuleTarget.create(
+        db=db,
+        data={
+            "name": "Fidesops Autogenerated Access Target",
+            "data_category": DataCategory("user.provided.identifiable").value,
+            "rule_id": access_rule.id,
+            "client_id": client.id,
+        },
+    )
+
+
+def autogenerate_erasure_policy(db: Session, client: ClientDetail) -> None:
+    """Create an autogenerated 'deletion' erasure policy, with an erasure rule attached,
+    targeting user.provided.identifiable data"""
+    logger.info(f"Creating autogenerated '{AUTOGENERATED_ERASURE_KEY}' policy")
+
+    erasure_policy = Policy.create_or_update(
+        db,
+        data={
+            "name": "Fidesops Autogenerated Erasure Policy",
+            "key": AUTOGENERATED_ERASURE_KEY,
+            "drp_action": DrpAction.deletion,
+            "client_id": client.id,
+        },
+    )
+
+    erasure_rule = Rule.create(
+        db,
+        data={
+            "name": "Fidesops Autogenerated Erasure Rule",
+            "action_type": ActionType.erasure.value,
+            "policy_id": erasure_policy.id,
+            "client_id": client.id,
+            "masking_strategy": {
+                "strategy": "null_rewrite",
```

Comment:
Just thinking `null_rewrite` might not be the best strategy as it relies on fields being nullable. Let's use `string_rewrite`

Reply:
good point, updated -